### PR TITLE
Screens 5.1 to 5.5 - UI Updates

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Commits
- [x] screens 5.1 to 5.5 - UI Updates

##### FreedCamp
- [x] [screens 5.1 to 5.5 - UI Updates](https://freedcamp.com/view/2956183/tasks/40590131)

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -335,6 +337,7 @@ module.exports = {
         h4: '1.125rem',
         h5: '1rem',
         h6: '0.875rem',
+        '6px': '0.375rem',
```
Comment (CrazyOptimist) [inline tailwind.config.js:340]: I'm asking designer to fix font size

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

**Review by CrazyOptimist — APPROVED** [review]

File: src/common/components/NFTCard/NFTCard.tsx [hunk 2]
```diff
@@ -233,16 +275,21 @@ const NFTCard = ({
...
+            isNFTPortfolio && 'px-3 pt-14px',
+          )}
         >
           <div className='flex items-center justify-between'>
             <span
-              className={`text-sm md:text-base font-medium ${
-                variant === 'portfolio'
-                  ? 'leading-6 text-h6 text-gray-77'
-                  : 'leading-none text-h5 text-gray-71'
+              className={`${
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/common/components/NFTCard/NFTCard.tsx]: fixed

File: src/common/components/NFTCard/NFTCard.tsx [hunk 3]
```diff
@@ -156,9 +186,18 @@ const NFTCard = ({
...
+                className='cursor-pointer'
+              />
             </Tooltip>
-            <span className='ml-2 mr-3 lg:mr-6'>1 of 1,000</span>
+            <span
+              className={cn(
+                isNFTPortfolio ? 'ml-5px mr-9px' : 'ml-2 mr-3 lg:mr-6',
+              )}
+            >
+              1 of 1,000
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/common/components/NFTCard/NFTCard.tsx]: fixed

File: tailwind.config.js [hunk 4]
```diff
@@ -23,7 +23,9 @@ module.exports = {
     },
     extend: {
       spacing: {
+        '1px': '0.063rem',
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply tailwind.config.js]: fixed

File: tailwind.config.js (hunk 1 above)
Comment (ngvtuan) [reply tailwind.config.js:339]: I think we can use that font size for now and then when our designer has been updated we can get back and update it.

Comment (CrazyOptimist) [reply tailwind.config.js:339]: Had a discussion and concluded exactly same as you thought. Approving this now.

**Review by CrazyOptimist — APPROVED** [review]
LGTM 💯

**Review by sanjanadhikari1 — APPROVED** [review]

**Pull request merged**